exonum/src/storage/proof_map_index/key.rs

```diff
@@ -259,52 +259,93 @@ impl PartialEq for DBKey {
 
 impl ::std::fmt::Debug for DBKey {
     fn fmt(&self, f: &mut ::std::fmt::Formatter) -> ::std::fmt::Result {
-        write!(f, "DBKey(")?;
-        for i in 0..self.len() {
-            write!(
-                f,
-                "{}",
-                match self.get(i) {
-                    ChildKind::Left => '0',
-                    ChildKind::Right => '1',
+        let mut bits = String::with_capacity(KEY_SIZE * 8);
```

Comment:
Are you sure that the capacity is sufficient? Seems like `|` symbols will require additional `KEY_SIZE` capacity.